src/app/styles/base/advanced-filter.scss

```diff
@@ -21,11 +21,11 @@
     }
 }
 
-bl-adv-filter-statepicker {
```

Comment:
I'd recommend moving stuff out of that file. If you need to modify a style there try to put in in the scss file next to the component instead. Its hard to find otherwise which style apply to what

Reply:
Fair point. I actually had a hard time finding this at first so makes sense to move to its own file (especially since I moved the html to its own file as well)